client/.eslintrc.json

```diff
@@ -11,20 +11,16 @@
   ],
   "plugins": ["react", "prettier", "flowtype"],
   "rules": {
-    "react/destructuring-assignment": [0],
-    "react/prefer-stateless-function": [0],
+    "camelcase": ["error", { "allow": ["^UNSAFE_"] }],
     "jsx-a11y/click-events-have-key-events": [0],
     "jsx-a11y/no-static-element-interactions": [0],
+    "max-len": ["error", 80],
     "no-unused-vars": [1],
-    "react/no-unused-state": [1],
-    "react/jsx-filename-extension": [
-      1,
-      {
-        "extensions": [".js", "jsx"]
-      }
-    ],
     "prettier/prettier": "error",
-    "max-len": ["error", 80],
-    "camelcase": ["error", { "allow": ["^UNSAFE_"] }]
+    "react/destructuring-assignment": [0],
+    "react/jsx-filename-extension": [1, { "extensions": [".js", "jsx"] }],
+    "react/no-unused-state": [1],
+    "react/prefer-stateless-function": [0]
+    // "sort-imports": [1]
```

Comment:
is this supposed to be commented out?

Reply:
Yeah, I think that its something good to have but right now its throwing too many warnings, so we'll worry about it after MVP.